webapp/client/src/components/FormFailureHeader.js

```diff
@@ -0,0 +1,41 @@
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+const FailureAlertArea = styled.div`
+  &.alert {
+    border-radius: 0;
+    border: 0;
+    padding: var(--spacing-05);
+  }
+
+  &.alert-danger {
```

Comment:
Why did you define 2 classes for the same component FailureAlertArea?

Reply:
That's a convention we committed on: If we have classes from the "old world" that get transferred to the react world, we keep the classes and just copy the styles of the classes over. So the old "FailureAlertArea" (although it wasn't called that) had these exact classes.